src/terraform/dss-ci-cd/dss/modules/aggregator/main.tf

```diff
@@ -6,6 +6,7 @@ resource "terraform_data" "image" {
     command = join(" ", [
       "gcloud builds submit econia",
       "--config cloudbuild.yaml",
+      "--timeout 10000",
```

Comment:
Why is this here? If necessary how about splitting out into a separate PR?

Reply:
Noting that this was because the aggregator wouldn't build for integration testing